DEVELOP.md

```diff
@@ -48,23 +48,44 @@ Most of the components within this project use Kotlin coroutines and `Flow`, so
 
 For modules that rely on state, try using `StateRepo` based classes. These automatically expose a `Flow` for state changes, and using `ColdStateRepo` cold flow behaviour can be used.
 
+## Branching
+
+Kaluga use [Gitflow](https://nvie.com/posts/a-successful-git-branching-model/).
+
+We use with the following branches:
+- `master`
+- `develop`
+- `release/*` (example: `release/0.1.2`)
+- `feature/[<issue_id>-]*` (example: `feature/345-bluetooth-scanning`)
+- `hotfix/[<issue_id>-]*` (example: `hotfix/678-navigation-crash`)
+
+When making a pull request fill in the template.
+
 ## Publishing
 
 ### Publishing process
 
-1. Bump version at `gradle/ext.gradle`:
+#### Version
+
+If you want to publish with a specific version string, you can override this value in `local.properties` (this file is already generated by your IDE normally), by setting `libraryVersion`:
 
 ```sh
-library_version = 'X.X.X'
+libraryVersion=0.1.0-special-build
```

Comment:
Now this variable name `libraryVersion` is a bit confusing in internal `library_version`. Maybe we can use more specific name, like `local_library_version`? Also will not mix camelCase with snake_case for globals.

Reply:
as for camelCase vs snake_case, just following what is there already in the relevant files.

for the name, I don't really understand the `local_` prefix, how is it "local"?

Reply:
Well, I can't get why we can't just override `library_version` inside `local.properties` but defined another `libraryVersion`

Reply:
Yes, I meant it comes from `local.properties`. Indeed I do prefer `kaluga.libraryVersion` more

Reply:
this is now updated, not just for the new property but existing.